xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -636,6 +636,7 @@ CVideoPlayer::CVideoPlayer(IPlayerCallback& callback)
 
   m_displayLost = false;
   m_error = false;
+  m_bCloseRequest = false;
```

Comment:
that needs to be set in Open(), not in ctor

Reply:
I've added it in `CVideoPlayer::OpenFile` as well.

Reply:
Looks good.
btw: I though you decided not to init in ctor anymore.

Reply:
I'm always a bit unsure how to deal with changes to existing code.
As `m_bAbortRequest` and `m_error` are initialized in the ctor as well, I thought I'd do the same for consistency.
I prefer consistency in existing files.